### ISSUE TITLE
Traefik breaks with bad ingress definition with no recovery

### ISSUE THREAD

**Issue by ssgtmas** [containous/traefik#3953]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
Using k8s (1.10.x) and traefik 1.6.6 on barehardware.
(Same thing happened with latest available image as well)

1) Set up a simple deployment and service.
Create 3 ingress files, like this:
```yml
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: simpleserver
  namespace: default
spec:
  rules:
  - host: simpleserver.k8s.domain.com
    http:
      paths:
      - backend:
          serviceName: simpleserver-svc
          servicePort: 8080
        path: /AAAA
---
```
Run the curl against the newly created 'host', and it works just fine.
Now create a second ingress. Note the quote in path...
```yml
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: simpleserver2
  namespace: default
spec:
  rules:
  - host: simpleserver2.k8s.domain.com
    http:
      paths:
      - backend:
          serviceName: simpleserver-svc
          servicePort: 8080
        path: /AAAA-"0"
---
```
The first site still continue to work.  The second one will be unavailable, and there will be a parse error in the logs.

Now create the 3rd ingress:
```yml
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: simpleserver3
  namespace: default
spec:
  rules:
  - host: simpleserver3k8s.domain.com
    http:
      paths:
      - backend:
          serviceName: simpleserver-svc
          servicePort: 8080
        path: /BBBB
---
```

3rd site will be unavailable as well.
Now remove the second ingress. The 3rd ingress will still be unavailable. The problem persists until you restart traefik.

### What did you expect to see?

Traefik should either 1) handle quotes properly or 2) warn/error on second ingress and ignore that, yet handle and support subsequent ingresses.

### What did you see instead?

I could knock out the traefik's ingresss controller for the entire cluster by using the wrong characters in the path for everyone.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.7.0
Codename:     maroilles
Go version:   go1.11
Built:        2018-09-24_09:57:21AM
OS/Arch:      linux/amd64

```
### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
checkNewVersion = false
logLevel = "WARN"
defaultEntryPoints = ["http","https"]
InsecureSkipVerify = true

[entryPoints]
  [entryPoints.http]
  address = ":20080"

  [entryPoints.https]
  address = ":20443"
    [entryPoints.https.tls]

  [entryPoints.traefik-admin]
  address = "127.0.0.1:28080"

[ping]
entrypoint = "http"

[metrics]
  [metrics.prometheus]
    buckets=[0.1,0.3,1.2,5.0]
    entrypoint = "traefik-admin"
  [metrics.statistics]
    recentErrors = 10

[traefikLog]
  filePath = "/var/log/traefik.log"
  format = "json"

[lifeCycle]
  requestAcceptGraceTimeout = "25s"
  graceTimeOut = "15s"

[forwardingTimeouts]
  dialTimeout = "5s"
  responseHeaderTimeout = "60s"

[api]
  entryPoint = "traefik-admin"
  dashboard = true
  [api.statistics]
    recentErrors = 10

[kubernetes]

```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```json
{"level":"error","msg":"Near line 117 (last key parsed 'backends.simpleserver.XXXXXXXXXX'): expected '.' or ']' to end table name, but got '0' instead","time":"2018-09-28T18:51:01Z"}
```

**Comment by dtomcej** [comment]
Related to #3753.

The issue is that we assume that data coming from the k8s api is sanitized, when it really has only been linted via the yaml parser.

I need to extend #3753 to all user-defined sections of the ingress object to prevent this sort of thing:

- Host
- Path
- Service Name
- Service Port

**Comment by dtomcej** [comment]
Closed by #3964
